python/tank/platform/software_launcher.py

```diff
@@ -505,6 +506,15 @@ def get_standard_plugin_environment(self):
                 "Not setting SHOTGUN_PIPELINE_CONFIGURATION_ID."
             )
 
+        bundle_cache_fallback_paths = self.sgtk.pipeline_configuration.get_bundle_cache_fallback_paths()
+        if bundle_cache_fallback_paths:
+            env["SHOTGUN_BUNDLE_CACHE_FALLBACK_PATHS"] = os.pathsep.join(bundle_cache_fallback_paths)
```

Comment:
I would argue actually that logging when variables are set is not necessary, as the last line of this method does this
`self.logger.debug("Returning Plugin Environment: \n%s" % pprint.pformat(env))`